addons/l10n_mx/data/template/account.account-mx.csv

```diff
@@ -36,14 +36,13 @@
 "cuenta216_11","IMSS withholding tax","216.11.01","liability_current","False","l10n_mx.tag_credit_balance_account","Retención de impuestos del IMSS"
 "cuenta302_01","Heritage","302.01.01","equity","False","l10n_mx.tag_credit_balance_account","Patrimonio"
 "cuenta304_01","Profit from previous years","304.01.01","equity","False","l10n_mx.tag_credit_balance_account","Utilidad de ejercicios anteriores"
-"cuenta305_01","Profit for the year","305.01.01","equity","False","l10n_mx.tag_credit_balance_account","Utilidad del ejercicio"
 "cuenta401_01","Sales and/or services taxed at the general rate","401.01.01","income","False","l10n_mx.tag_credit_balance_account","Ventas y/o servicios gravados a la tasa general"
+"cuenta402_01","Returns, discounts or bonuses over sales and/or services at the general rate","402.01.01","income_other","False","l10n_mx.tag_debit_balance_account","Devoluciones, descuentos o bonificaciones sobre ventas y/o servicios a tarifa general"
 "cuenta501_01","Cost of sales","501.01.01","expense_direct_cost","False","l10n_mx.tag_debit_balance_account","Costo de venta"
+"cuenta503_01","Returns, discounts or bonuses over purchases","503.01.01","income_other","False","l10n_mx.tag_credit_balance_account","Devoluciones, descuentos o bonificaciones sobre compras"
```

Comment:
@mial-odoo This needs a functional approval from your side

Reply:
It is correct